tests/integration/query_graph/test_online_serving.py

```diff
@@ -89,6 +90,9 @@ def features_fixture(event_table, source_type):
     return features
 
 
+@pytest.mark.parametrize(
+    "source_type", SNOWFLAKE_SPARK_DATABRICKS, indirect=True  # skip unity catalog: DEV-3109
```

Comment:
Is this an issue that rarely occurs? I think we unconditionally skip the `test / test-integration-databricks (pull_request) Skipped` suite now, so `SNOWFLAKE_SPARK_DATABRICKS` is effectively the same as `SNOWFLAKE_SPARK`.

Reply:
I think this issue occurs deterministically. After we fixed other flaky tests.

Reply:
Oh I meant whether it's an issue that will be easily encountered by users, or more a corner case that rarely occurs in real life.

Reply:
I suspect we need to fix it. But it seems like `Feast` doesn't expect `np.float32` type (based on my rough understanding of the exception message show in the ticket). https://featurebyte.atlassian.net/browse/DEV-3109